src/loaders/utils/flinkAiModelsQuery.ts

```diff
@@ -0,0 +1,41 @@
+/** CCLoudResourceLoader Flink statement utils for AI Models */
+import { FlinkAIModel } from "../../models/flinkAiModel";
+import type { CCloudFlinkDbKafkaCluster } from "../../models/kafkaCluster";
+
+/**
+ * Generate the query to list all available Flink AI models for a given Flink catalog+database.
+ * @see https://docs.confluent.io/cloud/current/flink/reference/statements/show.html#flink-sql-show-models
```

Comment:
TIL about `@see` ! 
I think this needs curlies and an `@link`
https://jsdoc.app/tags-see